backend/terraform/modules/analyzer/modules/engine_cluster/main.tf

```diff
@@ -84,6 +84,12 @@ resource "aws_launch_template" "engine-template" {
     }
   }
 
+  metadata_options {
+    http_endpoint               = "enabled"
+    http_tokens                 = "required"
+    http_put_response_hop_limit = 3
```

Comment:
The hop limit had to be increased from the default because trying to access metadata from inside a docker container adds extra hops